.travis.yml

```diff
@@ -50,7 +50,12 @@ jobs:
       env:
         - TESTING_PROFILE=regression-integration-tests
       if: branch = master OR branch =~ /^release.*$/
-        
+
+    - stage: singularity-tests
```

Comment:
This is getting is own stage which means that Travis-CI will wait for all four coverage test jobs to complete before starting this job. You probably want to run this in parallel with the other coverage jobs since this is invoking the coverage profile anyway. 
![Screenshot from 2020-02-12 17-33-11](https://user-images.githubusercontent.com/1730679/74383908-1df5e280-4dbe-11ea-938a-76754708c0b3.png)